### PR TITLE
Run migration rules on PHP 8

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

**Comment by kubawerlos** [comment]
Return annotation for `__sleep` method could be removed when https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5718 is merged.

Other changes looks acceptable to me, but I'm open to sugestions. WDYT @keradus as it was your [idea](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5689#discussion_r626909103) to make this?

File: src/Fixer/PhpUnit/PhpUnitSizeClassFixer.php [hunk 1]
```diff
@@ -172,13 +172,13 @@ private function splitUpDocBlock($lines, Tokens $tokens, $docBlockIndex)
     }
 
     /**
-     * @param Line|Line[]|string $line
+     * @param Line $line
```
Comment (keradus) [inline src/Fixer/PhpUnit/PhpUnitSizeClassFixer.php:175]: could we put type in method declaration directly now?

File: src/Fixer/PhpUnit/PhpUnitSizeClassFixer.php [hunk 2]
```diff
@@ -172,13 +172,13 @@ private function splitUpDocBlock($lines, Tokens $tokens, $docBlockIndex)
...
      *
      * @return string
      */
     private function getSingleLineDocBlockEntry($line)
```
Comment (keradus) [inline src/Fixer/PhpUnit/PhpUnitSizeClassFixer.php:179]:
```suggestion
    private function getSingleLineDocBlockEntry(Line $line)
```

File: src/Fixer/PhpUnit/PhpUnitSizeClassFixer.php (hunk 1 above)
Comment (keradus) [reply src/Fixer/PhpUnit/PhpUnitSizeClassFixer.php:175]:
```suggestion
```

File: src/Fixer/PhpUnit/PhpUnitSizeClassFixer.php [hunk 3]
```diff
@@ -172,13 +172,13 @@ private function splitUpDocBlock($lines, Tokens $tokens, $docBlockIndex)
     }
 
     /**
-     * @param Line|Line[]|string $line
+     * @param Line $line
      *
```
Comment (keradus) [inline src/Fixer/PhpUnit/PhpUnitSizeClassFixer.php:176]:
```suggestion
```

**Review by keradus — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**